

On Mon, Sep 13, 2004 at 01:50:03AM -0700, Andrew Morton wrote:
>> Due to master.kernel.org being on the blink, 2.6.9-rc1-mm5 Is currently at
>>  http://www.zip.com.au/~akpm/linux/patches/2.6.9-rc1-mm5/
>> and will later appear at
>>  ftp://ftp.kernel.org/pub/linux/kernel/people/akpm/patches/2.6/2.6.9-rc1/2.6.9-rc1-mm5/
>> Please check kernel.org before using zip.com.au.

On Mon, Sep 13, 2004 at 07:25:30PM -0700, William Lee Irwin III wrote:
> The following 3 updates address various issues expressed to me in
> unrelated threads or messages, and while none of them are particularly
> pressing each does resolve a concern I've deemed valid.

I was informed that the vendor component of the copyright can't be
clobbered without more care, so this patch retains the older vendor,
updating it only to reflect the appropriate time period.

Index: mm5-2.6.9-rc1/kernel/pid.c
===================================================================

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/kernel/pid.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletion(-)

diff -puN kernel/pid.c~pidhashing-retain-older-vendor-copyright kernel/pid.c
--- 25/kernel/pid.c~pidhashing-retain-older-vendor-copyright	Tue Sep 14 16:41:02 2004
+++ 25-akpm/kernel/pid.c	Tue Sep 14 16:41:02 2004
@@ -1,7 +1,8 @@
 /*
  * Generic pidhash and scalable, time-bounded PID allocator
  *
- * (C) 2002-2004 William Irwin, Oracle
+ * (C) 2002-2003 William Irwin, IBM
+ * (C) 2004 William Irwin, Oracle
  * (C) 2002-2004 Ingo Molnar, Red Hat
  *
  * pid-structures are backing objects for tasks sharing a given ID to chain
_
